pdm/project/config.py

```diff
@@ -199,6 +200,11 @@ class Config(MutableMapping[str, str]):
         "pypi.verify_ssl": ConfigItem(
             "Verify SSL certificate when query PyPI", True, coerce=ensure_boolean
         ),
+        "pypi.ca_certs": ConfigItem(
+            "Path to a CA certificate bundle used for verifying the identity "
+            "of the PyPI server",
+            certifi.where(),
```

Comment:
This default value will show in the output of `pdm config`, which will be a bit noisy IMO. I wonder if it will be better if we just use None here and get the default path when used.

Reply:
> This default value will show in the output of `pdm config`, which will be a bit noisy IMO.

I agree 👍🏻

---

>. I wonder if it will be better if we just use None here and get the default path when used.

I have updated the code to do just that. I'm wondering though whether we should introduce something like `hide_default` into `ConfigItem` since especially in this case, `config.py` seems to be the right place for specifying the default (at least to me). What do you think?

Reply:
> whether we should introduce something like hide_default into ConfigItem since especially in this case

Good suggestion, but I think it is okay for now since `ca_certs` is the only one. And implementing such a behavior needs a bit refactor around the logic of retrieving the config value(you need to distinguish whether it comes from default or input). So thanks for your work, it LGTM.

Reply:
Perfect, I will rebase it then 👍🏻